app/src/main/java/com/example/ana/exampleapp/MainActivity.java

```diff
@@ -165,7 +175,21 @@ public boolean onOptionsItemSelected(MenuItem item) {
      * @param view the {@link View} that calls the method
      */
     public void btnStart(View view) {
-        int pin = settings.getInt("pin", 0);
```

Comment:
why have you removed this? It seems to be for testing proposes, so please remove it!